pvcscale.yml

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
We should put this file inside the workloads directory to be consistent with the other workloads.